hack/install.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -eux
+
```

Comment:
you should check here if the command exists, and if it does which version it is. if the version match the expectation, just exit, otherwise proceed